README.md

```diff
@@ -47,6 +47,16 @@ To run features with a specific tag run
 
 To include or exclude tags see [the cucumber documentation](https://github.com/cucumber/cucumber/wiki/Tags#running-a-subset-of-scenarios)
 
+## Tags
```

Comment:
Note, we don't _think_ we have any `skip` tags in our repository's at the moment so another approach is we just remove the `skip` tag from the jenkins jobs.

We have also spotted a `wip` tag being used in the legacy functional tests which is being excluded by jenkins at the moment
https://github.gds/gds/digitalmarketplace-jenkins/blob/26c0796acf3e3fed9611c718c1e47fe92e36a261/job_definitions/functional_tests_legacy.yml